README.md

```diff
@@ -16,28 +17,51 @@ This allows messaging clients to verify the identity of
 users automatically, and prevents malicious/compromised servers from
 hijacking secure communications without getting caught.
 
+This repository provides a [Golang](https://golang.org) implementation of the 
+CONIKS system. The implementation consists of a library, described in the 
+following section, and a standalone CONIKS-server and -client using this library.
+
 ## Golang Library
+
 The packages in this library implement the various components of the CONIKS
 system and may be imported individually.
 
-- ``bots``: Registration proxies for user account verification
-- ``client``: A reference implementation of a CONIKS client
-- ``crypto``: Cryptographic algorithms and operations
-- ``keyserver``: A reference implementation of a CONIKS key server
-- ``merkletree``: Merkle prefix tree and related data structures
-- ``utils``: Utility functions
-- ``protocol``: CONIKS protocols implementation/library
-- ``storage``: Hooks for persistent storage backend (currently unused)
+- `bots`: Registration proxies for user account verification
+- `client`: A reference implementation of a CONIKS client
+- `crypto`: Cryptographic algorithms and operations
+- `keyserver`: A reference implementation of a CONIKS key server
+- `merkletree`: Merkle prefix tree and related data structures
+- `utils`: Utility functions
+- `protocol`: CONIKS protocols implementation/library
+- `storage`: Hooks for persistent storage backend (currently unused)
 
 ## Disclaimer
+
 Please keep in mind that this CONIKS library is under active development.
 The repository may contain experimental features that aren't fully tested.
 We recommend using a [tagged release](https://github.com/coniks-sys/coniks-go/releases).
 
-##Documentation
+## Installation
+
+The preferred way to install CONIKS reference server or client implementation
+is by downloading the [latest release](releases/latest).
```

Comment:
I see yeah, it would be cool if we could just include the binary, but you're right, maybe we shouldn't do that yet. So, maybe we should clarify in the README that the way to install it is from source right now? Might make it a bit easier even, since then the flow is just `go get; go install` for the reference implementations.